packages/web-api/src/types/request/slackLists.ts

```diff
@@ -0,0 +1,910 @@
+import type { RichTextBlock } from '@slack/types';
```

Comment:
nit: This is a big nit, so I'm going to leave it up to you on whether to do it. It would be nice to alphabetically order the exports for each section (field types, schema, response-only). This makes finding the types a little easier for future maintainers, but sometimes you can't alphabetically order the exports when order matters (one export uses another).